language/en/units.json

```diff
@@ -1178,6 +1179,7 @@
 			"raptor_matriarch_fire": "The Mother of the Flamethrower class!",
 			"raptor_matriarch_spectre": "The Mother of the Spectre class!",
 			"chip": "Chip",
+			"comeffigy": "Transposes with Commander upon death; build one only",
```

Comment:
Make it so you _can't_ build more than one - if an effigy exists, disable the build tile. Adding the reason a build tile is disabled (e.g. wind turbines on no wind maps) is a separate, outstanding issue, so don't feel compelled to fix it in this PR.

Reply:
Building a new effigy deletes the old effigy. This is intentional. Building additional effigies for a metalcost is a balancing measure. Infinite respawning intended.

Then again, deleting old effigy isn't working properly right now. Might be simpler to maxthisunit or somesuch instead.

Reply:
I don't quite follow - if the player is not supposed to build more than one at a time, disable the build tile, if the player is supposed to build more than one, don't say "build one only".

Reply:
There isn't one "effigy" unit, there's 5 variants. The gadget handles hotswapping them as commanders evolve (Although that's kinda broken right now). Is there a way for me to disable build tiles for all 5 unitdefs based on if any one of them exists at a time?

Reply:
All unintentional edgecases (I'm aware of) have been resolved. It's all handled in the gadget because maxthisunit won't work for multiple unitdefs.